src/js/popper.js

```diff
@@ -0,0 +1,1210 @@
+      f = e.offsetWidth - d,
+      h = e.offsetHeight - a;
+    if (f || h) {
+      var u = t(e);
+      (f -= m(u, "x")), (h -= m(u, "y")), (p.width -= f), (p.height -= h);
+    }
+    return g(p);
+  }
+  function b(e, o) {
+    var i = 2 < arguments.length && void 0 !== arguments[2] && arguments[2],
+      p = r(10),
+      s = "HTML" === o.nodeName,
+      d = u(e),
+      a = u(o),
+      l = n(e),
+      m = t(o),
+      h = parseFloat(m.borderTopWidth, 10),
+      c = parseFloat(m.borderLeftWidth, 10);
+    i && s && ((a.top = ee(a.top, 0)), (a.left = ee(a.left, 0)));
+    var b = g({
+      top: d.top - a.top - h,
+      left: d.left - a.left - c,
+      width: d.width,
+      height: d.height,
+    });
+    if (((b.marginTop = 0), (b.marginLeft = 0), !p && s)) {
+      var w = parseFloat(m.marginTop, 10),
+        y = parseFloat(m.marginLeft, 10);
+      (b.top -= h - w),
+        (b.bottom -= h - w),
+        (b.left -= c - y),
+        (b.right -= c - y),
+        (b.marginTop = w),
+        (b.marginLeft = y);
+    }
+    return (
+      (p && !i ? o.contains(l) : o === l && "BODY" !== l.nodeName) &&
+        (b = f(b, o)),
+      b
+    );
+  }
+  function w(e) {
+    var t = 1 < arguments.length && void 0 !== arguments[1] && arguments[1],
+      o = e.ownerDocument.documentElement,
+      n = b(e, o),
+      i = ee(o.clientWidth, window.innerWidth || 0),
+      r = ee(o.clientHeight, window.innerHeight || 0),
+      p = t ? 0 : l(o),
+      s = t ? 0 : l(o, "left"),
+      d = {
+        top: p - n.top + n.marginTop,
+        left: s - n.left + n.marginLeft,
+        width: i,
+        height: r,
+      };
+    return g(d);
+  }
+  function y(e) {
+    var n = e.nodeName;
+    if ("BODY" === n || "HTML" === n) return !1;
+    if ("fixed" === t(e, "position")) return !0;
+    var i = o(e);
+    return !!i && y(i);
+  }
+  function E(e) {
+    if (!e || !e.parentElement || r()) return document.documentElement;
+    for (var o = e.parentElement; o && "none" === t(o, "transform"); )
+      o = o.parentElement;
+    return o || document.documentElement;
+  }
+  function v(e, t, r, p) {
+    var s = 4 < arguments.length && void 0 !== arguments[4] && arguments[4],
+      d = { top: 0, left: 0 },
+      l = s ? E(e) : a(e, i(t));
+    if ("viewport" === p) d = w(l, s);
+    else {
+      var f;
+      "scrollParent" === p
+        ? ((f = n(o(t))),
+          "BODY" === f.nodeName && (f = e.ownerDocument.documentElement))
+        : "window" === p
+        ? (f = e.ownerDocument.documentElement)
+        : (f = p);
+      var m = b(f, l, s);
+      if ("HTML" === f.nodeName && !y(l)) {
+        var h = c(e.ownerDocument),
+          g = h.height,
+          u = h.width;
+        (d.top += m.top - m.marginTop),
+          (d.bottom = g + m.top),
+          (d.left += m.left - m.marginLeft),
+          (d.right = u + m.left);
+      } else d = m;
+    }
+    r = r || 0;
+    var v = "number" == typeof r;
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5731813)